app/menus/menus/edit.js

```diff
@@ -52,7 +51,7 @@ module.exports = function () {
         label: 'Preferences...',
         accelerator: accelerators.preferences,
         click() {
-          shell.openItem(confPath);
+          openConfig();
```

Comment:
Can you use the same `onpenConfig()` function here: https://github.com/zeit/hyper/blob/3c1f359198a71c6946f326bfda3e53f612ade23e/app/menus/menus/darwin.js#L21 ?